CHANGES.rst

```diff
@@ -14,6 +14,7 @@ Unreleased
   ``bulk_update_with_history`` instead of ``objects`` (gh-703)
 - Add optional ``manager`` argument to ``bulk_update_with_history`` to use instead of
   the default manager (gh-703)
+- Removed support for Django versions prior to 2.2
```

Comment:
Can you move this line up to the `Unreleased` section above?

Reply:
Also, can you append `(gh-652)` to the end of the line?

Reply:
Updated 👍